EIPS/eip-634.md

```diff
@@ -29,9 +31,15 @@ associate arbitrary key-value text.
 
 A new resolver interface is defined, consisting of the following method:
 
-    function text(bytes32 node, string key) constant returns (string text);
+```solidity
```

Comment:
Actually, what many other EIPs do when they claim an interface ID is to actually describe this as an interface (because then Solidity can generate the interface IDs):
```solidity
interface IERC634 {
    function text(....
}
```

This is only a suggestion.

Reply:
I've made that change, thanks! :)